websockets/chat-broker/README.md

```diff
@@ -3,11 +3,17 @@
 This is a different implementation of the
 [websocket chat example](https://github.com/actix/examples/tree/master/websockets/chat)
 
+This is based on 
```

Comment:
I think it's a bit weird to criticize the project in the very same repository, just in another folder.
Like, it will make reader wonder "If it's so lame, why won't you either fix or deprecate it?", which is not a good thing, especially because both examples have their reasons to exist (e.g. old one may be interesting to folks that don't want to use `actix-broker`).

Also, in the current form, it has two paragraphs referring to the `websocket chat example`; I think it makes much sense, I'd go with just one.

Reply:
You want me to >notice that this example is superior to the linked one
and to not criticize the chat eg. So I'll write just this in the 1st paragraph?

>This is a different implementation of the
[websocket chat example](https://github.com/actix/examples/tree/master/websockets/chat),
but it has includes: some refactoring, additional features, and requires
more dependancies.

Reply:
Yeah, I think that'd be more than enough.

Reply:
ok, updated it